webcompat/static/css/development/components/upload-form.css

```diff
@@ -92,6 +94,9 @@
   text-decoration: underline;
   transform: translateY(0);
   transition: transform .2s linear 0s;
+
+  /* workaround for mult dnd */
```

Comment:
@miketaylr this is part of the hack solution described at the bottom. originally the UI was designed to either:
* only have the file input field active or
* only have the remove button/link active

in order to make the 2nd drag/drop work, the design had to be updated to allow both fields to be interactive at the same time. otherwise the default behavior would be:
* user drags 2nd file over file input field
* user tries to click the Remove button/link => the dialog to upload a new file pops up
  * workaround was to put the Remove button on top